xskillscore/core/deterministic.py

```diff
@@ -286,3 +388,132 @@ def mae(a, b, dim, weights=None):
         dask="parallelized",
         output_dtypes=[float],
     )
+
+
+def mad(a, b, dim, weights=None):
+    """
+    Mean Absolute Deviation.
+
+    Parameters
+    ----------
+    a : xarray.Dataset or xarray.DataArray
+        Labeled array(s) over which to apply the function.
+    b : xarray.Dataset or xarray.DataArray
+        Labeled array(s) over which to apply the function.
+    dim : str, list
+        The dimension(s) to apply the mae along.
+    weights : xarray.Dataset or xarray.DataArray
+        Weights matching dimensions of ``dim`` to apply during the function.
+        If None, an array of ones will be applied (i.e., no weighting).
+
+    Returns
+    -------
+    xarray.Dataset or xarray.DataArray
+        Median Absolute Deviation.
+
+    See Also
+    --------
+    sklearn.metrics.median_absolute_error
+    xarray.apply_ufunc
+    https://en.wikipedia.org/wiki/Median_absolute_deviation
```

Comment:
I agree this is useful but can you put it in a References section
https://github.com/scipy/scipy/blob/master/scipy/stats/stats.py#L930